### DISCUSSION TITLE
How to make mermaid's icons appear

### DISCUSSION THREAD

**Discussion by ComputationTime** [imfing/hextra#208] [Q&A] [answered]
I'm new to Hugo and Hextra, and am trying to figure out how to make mermaid's icons appear. Any help will be appreciated!

**Comment by ComputationTime** [comment]
If this is answered, I am willing to submit a pull request to add to the docs if it's deemed useful

**Comment by imfing** [comment]
Sorry, I don't quite understand, what do you mean by "mermaid's icons"? Can you give some examples?

**Reply by ComputationTime** [reply]
From the mermaid documentation here http://mermaid.js.org/syntax/flowchart.html on the section named "Basic support for fontwesome":

```text
flowchart TD
    B["fa:fa-twitter for peace"]
    B-->C[fa:fa-ban forbidden]
    B-->D(fa:fa-spinner)
    B-->E(A fa:fa-camera-retro perhaps?)
```
  I tried using it as is with markdown (code used shown below) and the result was this
   <img src="https://github.com/imfing/hextra/assets/91295734/60d78e99-3555-4b81-99c6-b35369a8e28e">
```markdown
    ```mermaid
    flowchart TD
      B["fa:fa-twitter for peace"]
      B-->C[fa:fa-ban forbidden]
      B-->D(fa:fa-spinner)
      B-->E(A fa:fa-camera-retro perhaps?)
    ```
```

**Reply by imfing** [reply, marked as answer]
I see. Currently, you need to include the [Font Awesome CSS](https://cdnjs.com/libraries/font-awesome) manually to enable this for Mermaid.

1. Create a file `/layouts/partials/custom/head-end.html` under your site root directory
2. Add the following to this file:
```html
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" integrity="sha512-z3gLpd7yknf1YoNbCzqRKc4qyor8gaKU1qmn+CShxbuBusANI9QpRohGBreCFkKxLhei6S9CQXFEbbKuqLg0DA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```

Result:

![image](https://github.com/imfing/hextra/assets/5097752/c6625beb-b465-4de6-967a-0445b0cbe397)
